src/graphql/main/docs/README.md

```diff
@@ -0,0 +1,48 @@
+# Lightning Integration
+
+## Overview
+The API endpoints are
+
+testnet: `https://api.testnet.galoy.io/graphql`
+
+mainnet: `https://api.mainnet.galoy.io/graphql`
+
+## Authentication
+To get a new JWT:
+1. Use `userRequestAuthCode` to receive an auth code via SMS
+2. Call `userLogin` using the same phone number and auth code
+
+All other methods require a valid JWT set in the header as a bearer token - `Authorization: Bearer`
+
+## Curl requests
+
+### userRequestAuthCode
+
+```
+curl --location --request POST 'https://api.testnet.galoy.io/graphql' --header 'Content-Type: application/json' --data-raw '{"query":"mutation userRequestAuthCode ($input: UserRequestAuthCodeInput!) {\n    userRequestAuthCode (input: $input) {\n        errors {\n            message\n            path\n        }\n        success\n    }\n}","variables":{"input":{"phone":666397746837462800}}}'
```

Comment:
`'https://api.testnet.galoy.io/graphql'` could be converted as a env variable. 

ie: 
```
export URI=https://api.testnet.galoy.io/graphql
curl --location --request POST $URI ...
```

Reply:
for the phone number, maybe use a dummy number that still have the correct syntax. for instance: https://fakenumber.org/